Source/Scene/Cesium3DTileset.js

```diff
@@ -113,6 +133,11 @@ define([
      * @param {Boolean} [options.debugShowViewerRequestVolume=false] For debugging only. When true, renders the viewer request volume for each tile.
      * @param {Boolean} [options.debugShowGeometricError=false] For debugging only. When true, draws labels to indicate the geometric error of each tile
      * @param {ShadowMode} [options.shadows=ShadowMode.ENABLED] Determines whether the tileset casts or receives shadows from each light source.
+     * @param {Boolean} [options.skipLODs=true] Determines if level-of-detail skipping optimization should be used.
+     * @param {Number} [options.skipSSEFactor=10] Multiplier defining the minimum screen space error to skip when loading tiles.
+     * @param {Number} [options.skipLevels=1] Constant defining the minimum number of levels to skip when loading tiles.
```

Comment:
> Should this default to 2 for tilesets that have wide SSE differences between tiles?

What is the motivation for this? At 1, it will load at most every other level.